lua/npc.lua

```diff
@@ -110,6 +110,11 @@ function wesnoth.wml_actions.npc_bird_behavior(cfg)
 
 			-- Move the bird
 
+			if cfg.use_all_moves then
+				npc.moves = 0
+				npc.attacks_left=0
+			end
```

Comment:
I'm wondering if this fixes (or at least addresses) issue #8 in this repo at all?

Reply:
Doesn't address #8 specifically, rather this allows the macro to be used with birds that aren't on an idle_ai side (otherwise AI would be free to use birds to attack).

Reply:
OK, got it.